File: flask-mvc-starter-kit/app/models/University.py
from flask_sqlalchemy import SQLAlchemy
from app import db
class University(db.Model):
    __tablename__ = 'University'
    
    universityId = db.Column(db.Integer, primary_key=True)
    universityName = db.Column(db.String(255), nullable=False)
    uniCountry = db.Column(db.String(255), nullable=False)
    uniSede = db.Column(db.String(255), nullable=False)
    uniAdress = db.Column(db.String(255), nullable=False)

    def to_dict(self):
        return {
            "universityId": self.universityId,
            "universityName": self.universityName,
            "uniCountry": self.uniCountry,
            "uniSede": self.uniSede,
            'uniAdress': self.uniAdress
        }

File: src/main/java/Model/University.java
package Model;

/**
 *
 * @author kathe
 */
public class University {
    protected int universityId;
    protected String universityName;
    protected String uniCountry;
    protected String uniSede;
    protected String uniAdress;

    public University(int universityId, String universityName, String uniCountry, String uniSede, String uniAdress) {
        this.universityId = universityId;
        this.universityName = universityName;
        this.uniCountry = uniCountry;
        this.uniSede = uniSede;
        this.uniAdress = uniAdress;
    }

    public int getUniversityId() {
        return universityId;
    }

    public void setUniversityId(int universityId) {
        this.universityId = universityId;
    }

    public String getUniversityName() {
        return universityName;
    }

    public void setUniversityName(String universityName) {
        this.universityName = universityName;
    }

    public String getUniCountry() {
        return uniCountry;
    }

    public void setUniCountry(String uniCountry) {
        this.uniCountry = uniCountry;
    }

    public String getUniSede() {
        return uniSede;
    }

    public void setUniSede(String uniSede) {
        this.uniSede = uniSede;
    }

    public String getUniAdress() {
        return uniAdress;
    }

    public void setUniAdress(String uniAdress) {
        this.uniAdress = uniAdress;
    }
    
}
